sphinx_immaterial/cpp_domain_fixes.py

```diff
@@ -734,10 +738,17 @@ def _add_parameter_documentation_ids(
             decl_text = desc_param_node.astext().strip()
             unique_decls.setdefault(decl_text, (i, desc_param_node))
         if not unique_decls:
+            all_params = {}
+            for sig_param_nodes in sig_param_nodes_for_signature:
+                all_params.update(sig_param_nodes)
             logger.warning(
-                "Invalid parameter name: %r", param_name, location=param_node
+                "Parameter name %r does not match any of the parameters "
+                "defined in the signature: %r",
```

Comment:
>sphinx_immaterial/cpp_domain_fixes.py:745:0: C0301: Line too long (102/90) (line-too-long)

It looks like this line was split to avoid the line-length mandated in black. You could remove that disabling flag in pylintrc file.

Reply:
Done.